packages/citizen-sdk/src/sdks/viem-claim-sdk.ts

```diff
@@ -266,18 +266,12 @@ export class ClaimSDK {
   }
 
   /**
-   * Redirects the user through the face-verification flow.
+   * Redirects the user through the face-verification flow with Farcaster support.
    * @throws If face verification redirect fails.
    */
   private async fvRedirect(): Promise<void> {
-    const fvLink = await this.identitySDK.generateFVLink(false, this.rdu, 42220)
-    if (typeof window !== "undefined") {
-      window.location.href = fvLink
-    } else {
-      throw new Error(
-        "Face verification redirect is only supported in browser environments.",
-      )
-    }
+    // Use the new navigation method that handles Farcaster automatically
+    await this.identitySDK.navigateToFaceVerification(false, this.rdu, 42220)
```

Comment:
the content of navigatetoface should be here... it doesnt make sense to have a one line method calling another method

Reply:
didn't we agree that we wanted to prevent having to maintain same code in different places? @sirpy

Reply:
@vortex-hue but this could be included in the regular claim-flow, does not have to be a separate method